flask-api/flaskr/__init__.py

```diff
@@ -21,17 +21,18 @@ def create_app(test_config=None):
     app.config["DEBUG"] = True
     app.config['JWT_SECRET_KEY'] =  os.environ['FLASK_SECRET_KEY']
     app.config['JWT_TOKEN_LOCATION'] = ['cookies']
-    app.config['JWT_COOKIE_CSRF_PROTECT'] = True
+    app.config['JWT_COOKIE_CSRF_PROTECT'] = False

```

Comment:
This will need to be updated later whenever @SamTheBosssss is back to help me debug what is going on with this.

Reply:
Probably in another PR if it requires major changes.